src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -59,6 +99,17 @@ public void findFirstWithAge30UsingGuavaPredicate() {
 
         // TODO использовать FluentIterable
         Person person = null;
+        Predicate<Person> isAge30Checker = new Predicate<Person>() {
+            @Override
+            public boolean apply(Person person) {
+                return person.getAge() == 30;
+            }
+        };
+        Optional<Person> personOptional = FluentIterable.from(persons)
+                .firstMatch(isAge30Checker);
+        if (personOptional.isPresent()) {
+            person = personOptional.get();
+        }
```

Comment:
Lines 101, 108 and 110 could be done in one line with orNull() after firstMatch().